src/Microsoft.OpenApi.Readers/V3/OpenApiDocumentDeserializer.cs

```diff
@@ -21,10 +21,8 @@ internal static partial class OpenApiV3Deserializer
                 } /* Version is valid field but we already parsed it */
             },
             {"info", (o, n) => o.Info = LoadInfo(n)},
```

Comment:
are those deletions intended?

Reply:
Yes. the properties are in 3.1-specific so I moved them to the V3.1 deserializer